src/main/resources/endpoints.properties

```diff
@@ -0,0 +1,9 @@
+protocol.host.port=http://localhost:8080
```

Comment:
looks like here not just port.
maybe `protocol.host`?

Reply:
I meant the protocol + host + port so separated it with points 
if it is wrong maybe "base.url" would be better?
What does "protocol.host" mean? Is it protocol + host?